strings/language/swears.txt

```diff
@@ -0,0 +1,128 @@
+swearwords
+	fuck
+	piss
+	shit
+	ass
+	arse
+	butt
+	crap
+	woof
+	damn
+	hell
+
+notswearwords
+	bazinga
+	yass
+	arf
+	yee
+	awoo
+	bark
+	bork
+	bref
+	borf
+	crud
+	frick
+	frickin
+	darn
+	darnation
+	kupo
+	heck
+	heckin
+	hella
+	syndie
+	nukie
+	wolf
+	herp
+	tider
+	bee
+	beemom
+	traser
+	chump
+	dork
+	dorp
+	dungus
+	dringus
+	bringus
+	brongus
+	blingus
+	dweeb
+	dweeber
+	idiot
+	idjit
+	dope
+	dank
+	sec
+	buttsky
+	hotdog
+	fightme
+	owl
+	daddio
+	fred
+	fart
+	chief
+	heisenbutt
+	aaaaa
+	harner
+	pizza
+	pissa
+	pisspizza
+	bup
+	weed
+	wee
+	pee
+	goon
+	pubbie
+	puppie
+	pali
+	salb
+	meth
+	triplemeth
+	triplepiss
+	triplefuck
+	fucklemeth
+	chucklepiss
+	methlepiss
+	wonk
+	ghost
+	discount
+	guava
+	disco
+	helpme
+	please
+	imsorry
+	forgiveme
+	goodbye
+	hi
+	heyo
+	fffff
+	bloody
+	tekwar
+	pianzi
+	pianzo
+	gragg
+	gringus
+	vurdalk
+	vodralik
+	verdork
+	josh
+	jerkass
+	buttford
+	yeet
+	myke
+	shark
+	jenkem
+	stew
+	[rand(100,999)] [pick("Alpha","Bravo","Delta","Omega","Gamma","Zeta")]
```

Comment:
This won't work as you expect it to. If you want this to work consider using the mess in `string_picker.dm` which allows for more advanced string shenanigans in txt files.